account_invoice_production_lot/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Invoice Production Lots
+===========
+
+This module prints, for each (customer) invoice line, the delivered production
+lots.
+
+Usage
+=====
+
+Create a Sale Order, in field  "Create Invoice" set "On Delivery Order".
```

Comment:
This module should work whatever invoicing policy is set for sale order, doesn't it?